regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -72,6 +96,10 @@ var CommentReviewView = Backbone.View.extend({
     this.render();
   },
 
+  toggleSubmit: function() {
+    $('.submit-button').prop('disabled', function(i, v) { return !v; });
```

Comment:
Procedural updates like this might not mix well with the template rendering we're doing elsewhere in this view. AFAICT, If a user requests a comment download, then checks the "I agree" box, and then the download finishes, we're going to re-render the entire section, which will disable the submit button again. We could roll this update into the template, or only use the `render` method to update certain parts of the view.